scripts/swagger-docs.py

```diff
@@ -23,14 +24,18 @@
 with open(MDW_VERSION_FILE) as mdw_version_file:
     mdw_version = mdw_version_file.read().strip()
 
+with open(os.path.join(SWAGGER_DOCS_DIR, 'node_oas3.yaml')) as node_stream:
+    node_oas3 = yaml.safe_load(node_stream)
+    node_schemas = node_oas3['components']['schemas']
+
 for filepath in Path(SWAGGER_DOCS_DIR).glob("*.spec.yaml"):
     with open(filepath, 'r') as stream:
         data_loaded = yaml.safe_load(stream)
         schemas = {**schemas, **data_loaded['schemas']}
         paths = {**paths, **data_loaded['paths']}
 
 swagger['paths'] = paths
-swagger['components']['schemas'] = {**swagger['components']['schemas'], **schemas}
+swagger['components']['schemas'] = {**swagger['components']['schemas'], **schemas, **node_schemas}
```

Comment:
`node_schemas` have own schemas for `Channel`, `KeyBlock`, `OracleQuery`, `Status`. Looks like it overrides mdw's schemas.

Reply:
I propose to add a code removing schemas not referenced in other places, like this
```python
schemas = swagger['components']['schemas']
schemas_count = 0
while schemas_count != len(schemas):
    schemas_count = len(schemas)
    swagger_string = str(swagger)
    for name in list(schemas):
        if '#/components/schemas/' + name not in swagger_string:
            del schemas[name]
```
It needs to be run multiple times to remove a chain of schemas referencing each other, but not referenced in other places. It reduces the schemas count from 142 to 97. This way you don't need to import specific schemas, but there won't be extra schemas in the final document.